starter/js/atm.js

```diff
@@ -1,43 +1,57 @@
-//Begin with the document ready function
-
-    //Checking account deposit function
-
-      //On click of the depositChecking button
-
-        //Get value from the amountChecking input field
-
-        //Take that value and add it to the existing value in the checkingBalance div
-
-    //Checking account withdrawl funtion
-
-      //On click of the withdrawChecking button
-
-        //Get value from the amountChecking input field
-
-        // If that value is greater than the value in the account ignore that action
-        // In other words if this would put the account into a negative balance do not allow it
-
-        //Else subtract that value from the current amount in the checking account
-
-    //Savings account deposit function
-
-      //On click of the depositSavings button
-
-        //Get value from the amountSavings input field
-
-        //Take that value and add it to the existing value in the savingsBalance div
-
-    //Savings account withdraw funtion
-
-      //On click of the withdrawl button
-
-        //Get value from the amountSavings input field
-
-         //If that value is greater than the value in the account ignore that action
-         //In other words if this would put the account into a negative balance do not allow it
-
-         //Else subtract that value from the current amount in the savings account
-
-// Bonus-- get the two accounts to work with each other and allow for overdraft protection
-
-// Double Bonus-- This isnt very DRY.  Using the keyword "this" see if you can make the withdraw and deposit functions work for both accounts
+$(document).ready(function() {
+  $('#withdrawSavings').on('click', savingsWithdraw);
+  $('#depositChecking').on('click', checkingDeposit);
+  $('#withdrawChecking').on('click', checkingWithdraw);
+  $('#depositSavings').on('click', savingsDeposit);
+  checkZero();
+
+});
+
+function checkZero() {
+  if (+ $('#checkingBalance').text().replace('$', '') === 0) {
+    $('#checkingBalance').addClass('zero');
+  } else {
+    $('#checkingBalance').removeClass('zero');
+  };
+  if (+ $('#savingsBalance').text().replace('$', '') === 0) {
+    $('#savingsBalance').addClass('zero');
+  } else {
+    $('#savingsBalance').removeClass('zero');
+  };
+};
+
+function checkingDeposit() {
+  amount = +$('#amountChecking').val();
+  balance = +$('#checkingBalance').text().replace('$', '');
+
+  $('#checkingBalance').html(balance + amount);
+  checkZero();
+
+};
+
+function checkingWithdraw() {
+  amount = +$('#amountChecking').val();
+  balance = +$('#checkingBalance').text().replace('$', '');
+  if (amount <= balance) {
+    $('#checkingBalance').html(balance - amount);
+  };
```

Comment:
Is this returning a string or an integer? You will want to use `parseInt()` to turn the text value into an integer so when you do your subtraction, the math is correct.